lib/console-log.coffee

```diff
@@ -65,7 +65,9 @@ module.exports =
         editorLineCount = editor.getLastScreenRow()
         functionCheckValues = ['=>', 'function', '){', ') {']
         identifierCaseConfig = configProp 'console-log.identifierCase'
-        identifier = (
+        identifierPrefixConfig = configProp 'console-log.identifierPrefix'
+        identifierPrefix = if identifierPrefixConfig then identifierPrefixConfig + ' ' else ''
```

Comment:
Shouldn't it be up to the user to add a space?

Reply:
Indeed, I'll fix.